File: pinyougou_parent/pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import java.util.List;

import com.pinyougou.pojo.TbSeckillGoods;

public interface SeckillService {

	List<TbSeckillGoods> findAllFromRedis();

	TbSeckillGoods findFromRedis(Long id);

	void saveSeckillOrder(Long id, String userId);

}

File: pinyougou_parent/pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function($http){
	
	this.findAllSeckillGoods=function(){
		return $http.get("./seckill/findAll");
	}
	this.findOne=function(id){
		return $http.get("./seckill/findOne?id="+id);
	}
	this.saveSeckillOrder=function(id){
		return $http.get("./seckill/saveSeckillOrder?id="+id);
	}
	
})